.github/CC_SERVER_INFO.md

```diff
@@ -4,9 +4,13 @@ Ubuntu 20.04
 
 ##### MODULES:
 
+- [mod-1v1-arena](https://github.com/azerothcore/mod-1v1-arena)
 - [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
 - [mod-anticheat](https://github.com/azerothcore/mod-anticheat)
+- [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue)
+- [mod-arena-replay](https://github.com/azerothcore/mod-arena-replay)
 - [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
+- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)

```

Comment:
This is wrong, we are using the forked module at @ AC

Reply:
Done.